src/OpenSage.Game.Tests/Terrain/Roads/RoadAlignmentTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using OpenSage.Data;
+using OpenSage.Data.Map;
+using OpenSage.Terrain.Roads;
+using Xunit;
+using static OpenSage.Terrain.Roads.RoadTextureType;
+
+namespace OpenSage.Tests.Terrain.Roads
+{
+    public class RoadAlignmentTests
+    {
+        [Fact]
+        public void NoAlignmentChange()
+        {
+            var topology = LoadTopologyFromMapFile();
+
+            AssertCurveType(topology.Nodes[0], BroadCurve);
+            AssertCurveType(topology.Nodes[1], Straight);
+            AssertCurveType(topology.Nodes[2], Straight);
+            AssertCurveType(topology.Nodes[3], TightCurve);
+        }
+
+        [Fact]
+        public void AlignmentChange()
+        {
+            var topology = LoadTopologyFromMapFile();
+
+            AssertCurveType(topology.Nodes[0], Straight);
+            AssertCurveType(topology.Nodes[1], BroadCurve);
+            AssertCurveType(topology.Nodes[2], Straight);
+            AssertCurveType(topology.Nodes[3], BroadCurve);
+        }
+
+        [Fact]
+        public void CHI01()
```

Comment:
Based on the file size, it looks like the CHI01.map that you've included isn't the _actual_ CHI01.map? (I ask because I don't think we should directly include .map files from the original games).

Reply:
No, I removed everything from the map except the two road segments that cause the problem.